Sources/SwiftyTailwind/Downloader.swift

```diff
@@ -13,11 +13,17 @@ enum DownloaderError: LocalizedError {
      This error is thrown when the binary name cannot be determined.
      */
     case unableToDetermineBinaryName
+    case checksumIsIncorrect
+    case errorReadingFilesForChecksumValidation
     
     var errorDescription: String? {
         switch self {
         case .unableToDetermineBinaryName:
             return "We were unable to determine Tailwind's binary name for this architecture and OS."
+        case .checksumIsIncorrect:
+            return "We attempted 5 downloads of the binary but the checksum never matched. Aborting."
+        case .errorReadingFilesForChecksumValidation:
+            return "We were unable to read files for checksum validation. Aborting."
```

Comment:
I think the aborting part is unnecessary because the users will already perceive the abortion when they get the error:

```suggestion
            return "We were unable to read files for checksum validation."
```